charms/kfp-schedwf/src/components/pebble_component.py

```diff
@@ -45,8 +46,9 @@ def get_layer(self) -> Layer:
                         "override": "replace",
                         "summary": "scheduled workflow controller service",
                         "startup": "enabled",
-                        "command": "/bin/controller --logtostderr=true"
-                        " --namespace={self.namespace}",
+                        "command": f"/bin/controller --logtostderr=true"
+                        f" --namespace={self.namespace}"
+                        f" --logLevel={self.log_level}",
```

Comment:
Similar to persistence, I got:

```
2024-07-04T23:42:08.586Z [pebble] Service "controller" starting: /bin/controller --logtostderr=true --namespace=kubeflow --logLevel=info
2024-07-04T23:42:08.657Z [controller] time="2024-07-04T23:42:08Z" level=info msg="Location: UTC"
2024-07-04T23:42:08.657Z [controller] flag provided but not defined: -logLevel
```

After refreshing the image `juju refresh kfp-schedwf --resource oci-image=charmedkubeflow/scheduledworkflow:2.2.0-517f00c` the unit went to active.